zwave_js_server/model/node.py

```diff
@@ -539,6 +732,71 @@ async def async_get_highest_security_class(self) -> SecurityClass:
         assert data
         return SecurityClass(data["highestSecurityClass"])
 
+    async def async_test_powerlevel(
+        self, test_node_id: int, powerlevel: Powerlevel, test_frame_count: int
+    ) -> int:
+        """Send testPowerLevel command to Node."""
+        data = await self.async_send_command(
+            "test_powerlevel",
+            testNodeId=test_node_id,
+            powerlevel=powerlevel,
+            testFrameCount=test_frame_count,
+            require_schema=13,
+            wait_for_result=True,
+        )
+        assert data
+        return cast(int, data["framesAcked"])
+
+    async def async_check_lifeline_health(
+        self, rounds: Optional[int] = None
+    ) -> LifelineHealthCheckSummary:
+        """Send checkLifelineHealth command to Node."""
+        kwargs = {}
+        if rounds is not None:
+            kwargs["rounds"] = rounds
+        data = await self.async_send_command(
+            "check_lifeline_health",
+            require_schema=13,
+            wait_for_result=True,
+            **kwargs,
+        )
+        assert data
+        return LifelineHealthCheckSummary(data["summary"])
+
+    async def async_check_route_health(
+        self, target_node_id: int, rounds: Optional[int] = None
+    ) -> RouteHealthCheckSummary:
+        """Send checkRouteHealth command to Node."""
+        kwargs = {"targetNodeId": target_node_id}
+        if rounds is not None:
+            kwargs["rounds"] = rounds
+        data = await self.async_send_command(
+            "check_route_health",
+            require_schema=13,
+            wait_for_result=True,
+            **kwargs,
+        )
+        assert data
+        return RouteHealthCheckSummary(data["summary"])
+
+    def handle_test_powerlevel_progress(self, event: Event) -> None:
+        """Process a test powerelevel progress event."""
+        event.data["test_powerlevel_progress"] = TestPowerlevelProgress(
```

Comment:
```suggestion
    def handle_test_power_level_progress(self, event: Event) -> None:
        """Process a test power level progress event."""
        event.data["test_power_level_progress"] = TestPowerLevelProgress(
```

Reply:
this function name change is incorrect, the event is currently `test powerlevel progress`